src/main/java/com/codeofus/rent_a_park/services/PersonService.java

```diff
@@ -41,6 +51,7 @@ public Person updatePerson(Person person) {
     }
 
     @Transactional
+    @CacheEvict(value = "student", key = "#personId")
```

Comment:
Not sure how this works, but on line 45 there was a #person.id, and here you don't have the .

Reply:
the argument is personId and it is used as a key for caching, and in the other, the argument is entity person, and it takes person.id as the key